demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -502,6 +502,26 @@ def create_contribution_items_version_note(self):
                     f"{item_versions.get('latest_version', '')}\n"
                 )
 
+            self.contribution_items_version_note += (
+                "> **For the reviewer:**\n "
+                "> 1. Compare the code of this PR with the latest version of the pack, make sure to understand what are"
+                " the changes that the contributor intended to contribute, and **solve the conflicts in accordance**.\n"
+                "> 2. If improvements are needed, instruct the contributor to edit the code through the **Github "
+                "Code-Space** and **Not through the XSOAR UI**.\n"
+            )
+
+            self.contribution_items_version_note += (
+                f"> **For the Contributor:** {self.gh_user}\n "
```

Comment:
Is this shown on github? If so, we can add a `@` prefix and have it auto-tag both the contributor and the author.

Reply:
Yes, that is what I meant to do with the contributor's user.